.travis.yml

```diff
@@ -29,48 +32,165 @@ env:
+      env: MODULE='cassandra' ARGS='-Dtest=**/graphdb/astyanax/*'
+    -
+      <<: *STANDARD_TEST_JOB
+      env: MODULE='cassandra' ARGS='-Dtest=**/diskstorage/cassandra/embedded/*'
+    -
+      <<: *STANDARD_TEST_JOB
+      env: MODULE='cassandra' ARGS='-Dtest=***/cassandra/*,*/graphdb/embedded/*'
+    -
+      <<: *STANDARD_TEST_JOB
+      env: MODULE='hbase-parent/janusgraph-hbase-10' ARGS='-Dtest=**/diskstorage/hbase/*'
+    -
+      <<: *STANDARD_TEST_JOB
+      env: MODULE='hbase-parent/janusgraph-hbase-10' ARGS='-Dtest=**/graphdb/hbase/*'
+    -
+      <<: *STANDARD_TEST_JOB
+      env: MODULE='hbase-parent/janusgraph-hbase-098' ARGS='-Dtest=**/diskstorage/hbase/*'
+    -
+      <<: *STANDARD_TEST_JOB
+      env: MODULE='hbase-parent/janusgraph-hbase-098' ARGS='-Dtest=**/graphdb/hbase/*'
+    -
+      <<: *STANDARD_TEST_JOB
+      env: MODULE='cql' ARGS='-Dtest=**/diskstorage/cql/* -Dtest.skip.murmur=true'
+    -
+      <<: *STANDARD_TEST_JOB
+      env: MODULE='cql' ARGS='-Dtest=**/diskstorage/cql/* -Dtest.skip.byteorderedpartitioner=true -Dtest.skip.murmur-serial=true -Dtest.skip.murmur-ssl=true'
+    -
+      <<: *STANDARD_TEST_JOB
+      env: MODULE='cql' ARGS='-Dtest=**/graphdb/cql/* -Dtest.skip.murmur=true'
+    -
+      <<: *STANDARD_TEST_JOB
+      env: MODULE='cql' ARGS='-Dtest=**/graphdb/cql/* -Dtest.skip.byteorderedpartitioner=true -Dtest.skip.murmur-serial=true -Dtest.skip.murmur-ssl=true'
+    
+    - stage: test
+      env: COVERITY_ONLY=true
+      install: travis_wait mvn --quiet install -DskipTests=true -Dmaven.javadoc.skip=true --batch-mode --show-version;
+      script:
+        - echo "Building Docker image for Coverity analysis";
+          docker build -t janusgraph/analysis analysis;
+          echo "Running Coverity scan";
+          travis_wait 50 \
+            docker run --rm \
+                -v ${HOME}/.m2:/root/.m2 -v ${PWD}:/opt/janusgraph \
+                -e COVERITY_SCAN_TOKEN="${COVERITY_SCAN_TOKEN}" \
+                -e COVERITY_EMAIL="${COVERITY_EMAIL}" \
+                -i janusgraph/analysis;
+      after_success:
+        # Upload test coverage reports to Codecov
+        - bash <(curl -s https://codecov.io/bash);
+      if: branch = coverity_scan
+
+    - stage: documentation
+      install: docker build -t doc-site:mkdocs -f docs.Dockerfile .
+      script:
+        - echo "Updating configuration markdown";
+          mvn --quiet clean install -DskipTests=true -pl janusgraph-doc -am;
+        - docker run --rm -v $PWD:/mkdocs doc-site:mkdocs mkdocs build
+    
+    - stage: deploy
+      install: skip
+      script: skip
+      before_deploy:
+        - echo "Updating configuration markdown";
+          mvn --quiet clean install -DskipTests=true -pl janusgraph-doc -am;
+        - echo "Download documentation generator";
+          echo https://github.com/containous/structor/releases/download/v1.3.2/structor_linux-amd64 | wget --output-document=$GOPATH/bin/structor -i -;
```

Comment:
I recommend to use our installation script:
```bash 
curl -sfL https://raw.githubusercontent.com/containous/structor/master/godownloader.sh | bash -s -- -b $GOPATH/bin v1.4.0
```
https://github.com/containous/structor#download--ci-integration

Reply:
Works perfectly.